Makefile

```diff
@@ -9,7 +9,6 @@ export COMPOSE_PROJECT_NAME := proximis
 
 # Create configuration files needed by the environment
 SETUP_ENV := $(shell (test -f $(SELF_DIR).env || cp $(SELF_DIR).env.dist $(SELF_DIR).env))
-SETUP_SERVER := $(shell (test -f $(SELF_DIR)nginx/servers/proximis.conf || cp $(SELF_DIR)nginx/servers/proximis.conf.dist $(SELF_DIR)nginx/servers/proximis.conf))
```

Comment:
Why do you remove this line?
It can be useful for people using the environment without  their own configuration file.